Library/Homebrew/cmd/install.rb

```diff
@@ -79,20 +79,95 @@
+                     "If `super` is passed, use superenv even if the formula specifies the "\
+                     "standard build environment."
+      switch "--ignore-dependencies",
+        description: "Skip installing any dependencies of any kind. If they are not already "\
+                     "present, the formula will probably fail to install."
+      switch "--only-dependencies",
+        description: "Install the dependencies with specified options but do not install the "\
+                     "specified formula."
+      flag "--cc=",
+        description: "Attempt to compile using provided <compiler>. <compiler> should be the "\
+                     "name of the compiler's executable, for instance `gcc-7` for GCC 7. "\
+                     "In order to use LLVM's clang, use `llvm_clang`. To specify the "\
+                     "Apple-provided clang, use `clang`. This parameter will only accept "\
+                     "compilers that are provided by Homebrew or bundled with macOS. "\
+                     "Please do not file issues if you encounter errors while using this flag."
+      switch "-s", "--build-from-source",
+        description: "Compile the specified <formula> from source even if a bottle is provided. "\
+                     "Dependencies will still be installed from bottles if they are available."
+      switch "--force-bottle",
+        description: "Install from a bottle if it exists for the current or newest version of "\
+                     "macOS, even if it would not normally be used for installation."
+      switch "--include-test",
+        description: "Install testing dependencies required to run `brew test`."
+      switch "--devel",
+        description: "If <formula> defines it, install the development version."
+      switch "--HEAD",
+        description: "If <formula> defines it, install the HEAD version, aka master, trunk, unstable."
+      switch "--fetch-HEAD"
+      switch "--keep-tmp",
+        description: "Dont delete the temporary files created during installation."
+      switch "--build-bottle",
+        description: "Prepare the formula for eventual bottling during installation."
+      switch :force,
+        description: "Install without checking for previously installed keg-only or "\
+                     "non-migrated versions."
+      switch :verbose,
+        description: "Print the verification and postinstall steps."
+      switch "--display-times",
+        description: "Print install times for each formula at the end of the run."
+      switch "-i", "--interactive",
+        description: "Download and patch <formula>, then open a shell. This allows the user to "\
+                     "run `./configure --help` and otherwise determine how to turn the software "\
+                     "package into a Homebrew package."
+      switch "-g", "--git",
+        description: "Create a Git repository, useful for creating patches to the software."
+
+      ARGV.formulae.each do |f|
```

Comment:
@MikeMcQuaid If a non existent formula name is passed, there would be an exception due to the way `ARGV.formulae` is handled. (Hence some tests in `install_spec` would fail)

Reply:
@GauthamGoli Yup, these exceptions are handled by the `rescue` at the bottom of the `install` method.